### PR TITLE
linux: add Sles 12 and 15 support and switch centos8 to rocky8

### PR DESCRIPTION
**Pull request by gdams**
closes: https://github.com/adoptium/installer/issues/420

### CODE REVIEW

**Review by karianna — APPROVED** [review]
LGTM - Did the PMC / someone end up documenting our officially supported platforms?

**Review by karianna — APPROVED** [review]
LGTM

File: linuxNew/jdk/suse/src/packageTest/java/packaging/SuseFlavours.java [hunk 1]
```diff
@@ -32,13 +32,13 @@
...
-		 * SUSE itself is not distributed via Docker Hub, so we cannot test against it.
-		 */
+		 * SLES: All supported versions, see https://www.suse.com/lifecycle.
+		*/
 		return Stream.of(
 			Arguments.of("opensuse/leap", "15.1"),
 			Arguments.of("opensuse/leap", "15.2"),
-			Arguments.of("opensuse/leap", "15.3")
+			Arguments.of("opensuse/leap", "15.3"),
+			Arguments.of("registry.suse.com/suse/sle15", "latest")
```
Comment (sophia-guo) [inline linuxNew/jdk/suse/src/packageTest/java/packaging/SuseFlavours.java:41]: Should also add sle12? `Arguments.of("registry.suse.com/suse/sle12", "latest")`

Comment (gdams) [reply linuxNew/jdk/suse/src/packageTest/java/packaging/SuseFlavours.java:41]: I tried adding sles12 but I needed a subscription to even run zypper update.

Comment (sophia-guo) [reply linuxNew/jdk/suse/src/packageTest/java/packaging/SuseFlavours.java:41]: Got it. We probably need to add some comment or open an issue to make a note of that. As this means we release installer on sles12 without any tests.

**Pull request merged**